dev-server/ui-playground/components/PatternPage.js

```diff
@@ -110,6 +103,8 @@ export function PatternExamples({ children, title }) {
  * @typedef PatternExampleProps
  * @prop {import("preact").ComponentChildren} children - Example source
  * @prop {string} details - Details about the pattern example
+ * @prop {Object} [style] - Optional additional inline styles to apply to the
```

Comment:
Pattern library component addition: Allows local inline styles when rendering an example pattern. I'm using this to set background color in some example patterns (some of the color palettes for buttons are intended for use on particular backgrounds).